R/module_filter_manager.R

```diff
@@ -111,109 +127,91 @@ filter_manager_srv <- function(id, filtered_data_list, filter) {
     }
     filtered_data_list <- flatten_nested(filtered_data_list)
 
-    # global list of slices (all available teal_slice)
-    slices_global <- reactiveVal(filter)
+    # # flatten nested alternative
+    # filtered_data_list <- unlist(filtered_data_list)
+    # names(filtered_data_list) <- sub("(.+)\\.(.+$)", "\\2", names(filtered_data_list))
```

Comment:
@gogonzo 
May I bother you for an opinion on this?

Reply:
Comment to remove. I prefer your solution with the recursive call